### PR TITLE
Added a Store Content Related URL

### PR DESCRIPTION
**Pull request by joinimran**
As the user has mentioned that to access the content of the store required URL was missing. I have updated the URL.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8874

Reset of the URL's mentioned in the comment doesn't have any reference in MS docs and previous version document. So as of now, I have dropped them. 

Thanks.
Imran.

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/privacy/manage-windows-20H2-endpoints.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ The following methodology was used to derive these network endpoints:
...
 |Microsoft Store|||[Learn how to turn off traffic to all of the following endpoint(s).](manage-connections-from-windows-operating-system-components-to-microsoft-services.md#26-microsoft-store)|
 ||The following endpoint is used to download image files that are called when applications run (Microsoft Store or Inbox MSN Apps). If you turn off traffic for these endpoints, the image files won't be downloaded, and apps cannot be installed or updated from the Microsoft Store. Additionally, the Microsoft Store won't be able to revoke malicious apps and users will still be able to open them.|HTTPS|img-prod-cms-rt-microsoft-com.akamaized.net|
+||The following endpoint is needed to load the content in Microsoft Store App.|HTTPS|livetileedge.dsx.mp.microsoft.com|
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-20H2-endpoints.md:88]:
```suggestion
||The following endpoint is needed to load the content in the Microsoft Store app.|HTTPS|livetileedge.dsx.mp.microsoft.com|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Comment by Gental-Giant** [comment]
#sign-off

**Pull request merged**